From 0bd35016dd8c4e751ea208f5e813456a85070c56 Mon Sep 17 00:00:00 2001
From: Felipe Contreras <felipe.contreras@gmail.com>
Date: Sun, 4 Jul 2010 16:31:47 +0300
Subject: [PATCH 12/94] staging: ti dspbridge: remove contributor leftovers

The old history (before git) had these two character ids that never
stood any chance to identify anybody.

Nobody from that list was moved to the good list of contributors, so
let's get rid of it.

Signed-off-by: Felipe Contreras <felipe.contreras@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---
 .../staging/tidspbridge/Documentation/CONTRIBUTORS |   37 --------------------
 1 files changed, 0 insertions(+), 37 deletions(-)

diff --git a/drivers/staging/tidspbridge/Documentation/CONTRIBUTORS b/drivers/staging/tidspbridge/Documentation/CONTRIBUTORS
index b40e7a6..86f5787 100644
--- a/drivers/staging/tidspbridge/Documentation/CONTRIBUTORS
+++ b/drivers/staging/tidspbridge/Documentation/CONTRIBUTORS
@@ -43,40 +43,3 @@ Please keep the following list in alphabetical order.
 	Armando Uribe de Leon
 	Nischal Varide
 	Wenbiao Wang
-
-
-
-The following list was taken from file Revision History, if you recognize your
-alias or did any contribution to the project please let us now, so we can
-proper credit your work.
-
-	ag
-	ap
-	cc
-	db
-	dh4
-	dr
-	hp
-	jg
-	kc
-	kln
-	kw
-	ge
-	gv
-	map
-	mf
-	mk
-	mr
-	nn
-	rajesh
-	rg
-	rr
-	rt
-	sb
-	sg
-	sh
-	sp
-	srid
-	swa
-	vp
-	ww
-- 
1.7.0.4

